test/cql-pytest/test_materialized_view.py

```diff
@@ -196,13 +196,20 @@ def test_mv_is_not_null(cql, test_keyspace):
 # Refs #10851. The code used to create a wildcard selection for all columns,
 # which erroneously also includes static columns if such are present in the
 # base table. Currently views only operate on regular columns and the filtering
-# code assumes that. TODO: once we implement static column support for materialized
+# code assumes that. Once we implement static column support for materialized
 # views, this test case will be a nice regression test to ensure that everything still
 # works if the static columns are *not* used in the view.
-def test_filter_with_unused_static_column(cql, test_keyspace):
+# This test goes over all combinations of filters for partition, clustering and regular
+# base columns.
+def test_filter_with_unused_static_column(cql, test_keyspace, scylla_only):
```

Comment:
If I understand correctly, the "scylla_only" is only for the v=44 case?
I guess it's ok. This test doesn't need to be the place where we document what is the exact difference between Scylla and Cassandra. But if we did want to do it, we could create two separate tests - one for the v = 44 case and the second for the others (which also works on Cassandra).

Reply:
Yes, I just decided not to split into two cases just for the sake of testing Cassandra on this path, since the semantics of view filtering are quite clear.